lib/simulator-management.js

```diff
@@ -20,7 +23,17 @@ const INSTALL_DAEMON_CACHE = 'com.apple.mobile.installd.staging';
  */
 async function createSim (caps) {
   const appiumTestDeviceName = `appiumTest-${UUID.create().toString().toUpperCase()}-${caps.deviceName}`;
-  const udid = await createDevice(appiumTestDeviceName, caps.deviceName, caps.platformVersion);
+  let udid;
+  try {
+    const platformVersion = `${isTvOS(caps.platformName) ? SIM_RUNTIME_NAME_TVOS : SIM_RUNTIME_NAME_IOS}-${caps.platformVersion.replace('.', '-')}`;
+    // platformVersion should be 'com.apple.CoreSimulator.SimRuntime.iOS-12-2'
+    // after installing over Xcode 10.2 once
+    udid = await createDevice(appiumTestDeviceName, caps.deviceName, platformVersion);
```

Comment:
I think the handling should rather be implemented in node-simctl. For clients of this lib SDK version should be transparent